packages/utils/src/lib/reports/__snapshots__/generate-md-report.unit.test.ts.snap

```diff
@@ -0,0 +1,204 @@
+|:--:|:--:|:--:|:--:|
+|🚨 <i>error</i>|Use design system components instead of classes|<code>list.component.ts</code>|400-200|
+|🚨 <i>error</i>|File size is 20KB too big|<code>list.component.ts</code>||
+
+
+</details>
+"
+`;
+
+exports[`auditDetailsIssues > should render complete section 1`] = `
+"#### Issues
+
+|Severity|Message|Source file|Line(s)|
+|:--:|:--:|:--:|:--:|
+|ℹ️ <i>info</i>|File \`index.js\` is 56Kb.|<code>index.js</code>||
+|⚠️ <i>warning</i>|Package license is has to be "MIT"|<code>package.json</code>|4|
+|🚨 <i>error</i>|no unused vars|<code>index.js</code>|400-200|
+"
+`;
+
+exports[`auditsSection > should render complete audit section 1`] = `
+"## 🛡️ Audits
+
+### No any (Eslint)
+
+🟩 <b>0</b> (score: 100)
+
+No unsafe any assignment
+
+### No let (Eslint)
+
+🟥 <b>5 errors</b> (score: 0)
+
+### Largest contentful paint (Lighthouse)
+
+🟨 <b>0</b> (score: 60)
+
+No unsafe any assignment [📖 Docs](https://web.dev/lcp)
+
+"
+`;
+
+exports[`generateMdReport > should render complete md report 1`] = `
+"# Code PushUp Report
+
+|🏷 Category|⭐ Score|🛡 Audits|
+|:--|:--:|:--:|
+|[Performance](#performance)|🟡 **61**|2|
+|[SEO](#seo)|🟢 **100**|1|
+|[PWA](#pwa)|🔴 **0**|1|
+
+## 🏷 Categories
+
+### Performance
+
+🟡 Score:  **61**
+- 🔴 Performance Group (_Lighthouse_)
+  - 🟨 [Largest Contentful Paint](#largest-contentful-paint-lighthouse) - **2,7 s**
+  - 🟩 [Fast Images](#fast-images-lighthouse) - **1**
+
+### SEO
+
+🟢 Score:  **100**
+- 🟥 [Website is crawlable](#website-is-crawlable-lighthouse) (_Lighthouse_) - **0**
+
+### PWA
+
+🔴 Score:  **0**
+- 🟩 [Splash Screen](#splash-screen-lighthouse) (_Lighthouse_) - **1**
+
+## 🛡️ Audits
+
+### Largest Contentful Paint (Lighthouse)
+
+<details>
+<summary>🟨 <b>2,7 s</b> (score: 67)</summary>
+
+#### Additional Information
+
+|Phase|% of LCP|Timing|
+|:--:|:--|--:|
+|TTFB|27%|620 ms|
+|Load Delay|25%|580 ms|
+|Load Time|41%|940 ms|
+|Render Delay|6%|140 ms|
+
+
+</details>
+
+This is the largest contentful element painted within the viewport. [Learn more about the Largest Contentful Paint element](https://developer.chrome.com/docs/lighthouse/performance/lighthouse-largest-contentful-paint/) [📖 Docs](https://web.dev/lcp)
+
+### Splash Screen (Lighthouse)
+
+🟩 <b>1</b> (score: 100)
+
+### Fast Images (Lighthouse)
+
+🟩 <b>1</b> (score: 97)
+
+### Website is crawlable (Lighthouse)
+
+🟥 <b>0</b> (score: 0)
+
+Search engines are unable to include your pages in search results if they don't have permission to crawl them. [Learn more about crawler directives](https://developer.chrome.com/docs/lighthouse/seo/is-crawlable/).
+
+### No explicit any (Eslint)
+
+<details>
+<summary>🟥 <b>63 errors</b> (score: 0)</summary>
+
+#### Issues
+
+|Severity|Message|Source file|Line(s)|
+|:--:|:--:|:--:|:--:|
+|ℹ️ <i>info</i>|File \`index.js\` is 56Kb.|<code>index.js</code>||
+|⚠️ <i>warning</i>|Package license is has to be "MIT"|<code>package.json</code>|4|
+|🚨 <i>error</i>|no unused vars|<code>index.js</code>|400-200|
```

Comment:
The italics and code formatting can be in Markdown now we no longer have HTML tables. Happy to leave this for another PR though.

```suggestion
|ℹ️ _info_|File \`index.js\` is 56Kb.|\`index.js\`||
|⚠️ _warning_|Package license is has to be "MIT"|\`package.json\`|4|
|🚨 _error_|no unused vars|\`index.js\`|400-200|
```